packages/plugin-inline/package.json

```diff
@@ -14,15 +14,18 @@
   "author": "Andrew Powell <andrew@shellscape.org>",
```

Comment:
you are never permitted to change package.json files per your instructions in `.charlie`